portal/code/company-portal/src/index.tsx

```diff
@@ -24,13 +24,13 @@ import { BrowserRouter } from 'react-router-dom';
 
 //runWithAdal(
 //  adalContext.AuthContext,
-UserService.initKeycloak(
-    () => {
+// UserService.initKeycloak(
```

Comment:
@vsamarora why did you comment out the keycloak login?

Reply:
updated file